src/Microsoft.Graph/Generated/Models/IdentityGovernance/WorkflowExecutionTrigger.cs

```diff
@@ -40,11 +40,7 @@ public class WorkflowExecutionTrigger : IAdditionalDataHolder, IBackedModel, IPa
         /// <param name="parseNode">The parse node to use to read the discriminator value and create the object</param>
         public static WorkflowExecutionTrigger CreateFromDiscriminatorValue(IParseNode parseNode) {
             _ = parseNode ?? throw new ArgumentNullException(nameof(parseNode));
-            var mappingValue = parseNode.GetChildNode("@odata.type")?.GetStringValue();
-            return mappingValue switch {
-                "#microsoft.graph.identityGovernance.timeBasedAttributeTrigger" => new TimeBasedAttributeTrigger(),
```

Comment:
Removing this mapping causes the dropping of the two types. cc @baywet

Reply:
The only reference to that type is in triggerAndScopeBasedConditions, which is not directly referenced but derives from workflowExecutionConditions. 

There's no way of directly accessing WorkflowExecutionTrigger and I'd in fact argue that this type itself should have been trimmed too.

Reply:
Just to confirm here. What were are saying is that since this is a derived type of a property from a derived type of another property, we would classify that as being to far off to be included?

Reply:
I'm guessing that if you get a workflowBase, for which the executionConditions property value is in fact a triggerAndScopeBasedConditions (derived from the declared type) of which the trigger property is in fact a timeBasedAttributeTrigger (derived from the declared type) you'd need it. Which makes this a breaking change.

I'm guessing this is cause by that change (no condition previously), I'm just worried that removing the condition would end up pulling too many types.
https://github.com/microsoft/kiota/blob/06e9deac11c68ff683ddb3f57504e7f655094829/src/Kiota.Builder/KiotaBuilder.cs#L1763

Let me look into it further